src/posts/blog/6-masks/masks.ts

```diff
@@ -0,0 +1,277 @@
+* **Breathability:** Varies
+* **Useful for:** Outdoor activities, especially exercising
+* **Benefits:** Looks pretty, easy to pull on and off, washable
+* **Downsides:** Minimal protection
+
+<figure>
+<img src=${poorfittingmasks} width="550", height="387"/>
+</figure>
+
+Cloth masks were a great stopgap solution in spring and summer 2020 when the world was struggling to produce enough PPE for healthcare workers.  However, that’s since changed, and you’re usually better off wearing a mask that brings more protection.  
+
+Cloth masks work by simple mechanical filtration.  As a result, they tend to not be that good at filtering the air you’re inhaling.  Tighter fabrics like denim tend to filter better than simple cotton t-shirt material, but they’re also harder to breathe through.  
+
+Some cloth masks are barely useful due to having thin fabric, poor fit, or both.  Masks in this category include bandanas, buffs, handkerchiefs, and t-shirts pulled over the nose. 
+
+Fitted cloth masks that are cut to match the shape of the face are a bit better, especially if they contain wire inserts to conform to the shape of the nose, or have head straps that let you secure the mask around the back of your head instead of just looping around your ears.  
+
+A handful of cloth masks come with PM2.5 filters, which are designed to more efficiently trap smaller particles.  These filters improve the performance of cloth masks to something similar to surgical masks, though it’s difficult to ensure that the air always goes through these filters.
+
+With caseloads at record levels, **cloth masks are not a good choice for indoor use**.  However, outdoor scenarios are low enough risk that a cloth mask should be adequate unless it’s a crowded situation such as a farmers market or a protest.
+
+However, there’s one situation where cloth masks still excel — outdoor exercise.  If you’re going to be running up a hill with other pedestrians around, a cloth mask is a great solution.  Unlike other masks, cloth masks are washable, so when you finish your run you can drop your sweaty cloth mask in the washing machine and use it again.  
+
+#### Shopping guide:  
+
+*   [Citizens of Humanity](https://citizensofhumanity.com/products/cotton-mask-5-pack) has great simple and inexpensive denim masks with head straps for ideal fit.  They’re extremely durable, high filtration efficiency, and still work well after ~20 washings.   
+*   [Rickshaw](https://www.rickshawbags.com/face-masks) also makes stylish masks with adjustable ear loops for improved fit. 
+*   [Electric Styles](https://www.electricstyles.com/collections/face-masks) makes cloth masks in a variety of patterns with removable PM2.5 filters
+
+<hr>
+
+## Surgical masks
+
+* **Protection of you:** Moderate
+* **Protection of others:** Good
+* **Fit quality:** Mediocre
+* **Breathability:** Good
+* **Useful for:** Outdoor activities
+* **Benefits:** Cheap
+* **Downsides:** Disposable, limited protection, can’t get wet. 
```

Comment:
I thought we agreed to remove the wet part? Maybe "can't be washed"?

Reply:
I'll go with "moisture degrades performance"

Reply:
> Could you update the graph based on published numbers? (specifically P100 + googles didn't make the cut)

Done